From a72e660c33f74f172b6e8e757d3efb8139231f4c Mon Sep 17 00:00:00 2001
From: Saurabh Sengar <saurabh.singh@xilinx.com>
Date: Sat, 24 Feb 2018 03:14:08 +0530
Subject: [PATCH 0290/1851] drm: xlnx: correcting data type for cursor with and
 height

commit 8dde87346bff5039b47b51f162cae0569d9e83fb from
https://github.com/Xilinx/linux-xlnx.git

Correcting data type to unsigned int from int

Signed-off-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_crtc.c | 8 ++++----
 drivers/gpu/drm/xlnx/xlnx_crtc.h | 4 ++--
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_crtc.c b/drivers/gpu/drm/xlnx/xlnx_crtc.c
index 44abb5b549a7..8fc0282fdbd9 100644
--- a/drivers/gpu/drm/xlnx/xlnx_crtc.c
+++ b/drivers/gpu/drm/xlnx/xlnx_crtc.c
@@ -128,10 +128,10 @@ uint32_t xlnx_crtc_helper_get_format(struct xlnx_crtc_helper *helper)
 	return format;
 }
 
-uint32_t xlnx_crtc_helper_get_cursor_width(struct xlnx_crtc_helper *helper)
+u32 xlnx_crtc_helper_get_cursor_width(struct xlnx_crtc_helper *helper)
 {
 	struct xlnx_crtc *crtc;
-	int width = XLNX_CRTC_MAX_HEIGHT_WIDTH, tmp;
+	u32 width = XLNX_CRTC_MAX_HEIGHT_WIDTH, tmp;
 
 	list_for_each_entry(crtc, &helper->xlnx_crtcs, list) {
 		if (crtc->get_cursor_width) {
@@ -143,10 +143,10 @@ uint32_t xlnx_crtc_helper_get_cursor_width(struct xlnx_crtc_helper *helper)
 	return width;
 }
 
-uint32_t xlnx_crtc_helper_get_cursor_height(struct xlnx_crtc_helper *helper)
+u32 xlnx_crtc_helper_get_cursor_height(struct xlnx_crtc_helper *helper)
 {
 	struct xlnx_crtc *crtc;
-	int height = XLNX_CRTC_MAX_HEIGHT_WIDTH, tmp;
+	u32 height = XLNX_CRTC_MAX_HEIGHT_WIDTH, tmp;
 
 	list_for_each_entry(crtc, &helper->xlnx_crtcs, list) {
 		if (crtc->get_cursor_height) {
diff --git a/drivers/gpu/drm/xlnx/xlnx_crtc.h b/drivers/gpu/drm/xlnx/xlnx_crtc.h
index b3c7056fd179..9ab57594aba8 100644
--- a/drivers/gpu/drm/xlnx/xlnx_crtc.h
+++ b/drivers/gpu/drm/xlnx/xlnx_crtc.h
@@ -54,8 +54,8 @@ u64 xlnx_crtc_helper_get_dma_mask(struct xlnx_crtc_helper *helper);
 int xlnx_crtc_helper_get_max_width(struct xlnx_crtc_helper *helper);
 int xlnx_crtc_helper_get_max_height(struct xlnx_crtc_helper *helper);
 uint32_t xlnx_crtc_helper_get_format(struct xlnx_crtc_helper *helper);
-uint32_t xlnx_crtc_helper_get_cursor_width(struct xlnx_crtc_helper *helper);
-uint32_t xlnx_crtc_helper_get_cursor_height(struct xlnx_crtc_helper *helper);
+u32 xlnx_crtc_helper_get_cursor_width(struct xlnx_crtc_helper *helper);
+u32 xlnx_crtc_helper_get_cursor_height(struct xlnx_crtc_helper *helper);
 
 struct xlnx_crtc_helper *xlnx_crtc_helper_init(struct drm_device *drm);
 void xlnx_crtc_helper_fini(struct drm_device *drm,
-- 
2.31.1

